Makefile

```diff
@@ -380,7 +380,7 @@ unittest: all
 runtest: all
 	@$(MAKE) -C $(srcdir)/tests TESTARG="$(TESTARG)" RUNTESTARG="$(RUNTESTARG)" test_run
 
-bench: all
+bench: all $(objdir)/misc/bench
```

Comment:
Ok, then it fails when we run `make bench`, right?

Reply:
Yes, right.